example/ViewPagerExample.js

```diff
@@ -125,7 +127,7 @@ export default class ViewPagerExample extends React.Component<*, State> {
           orientation="horizontal"
           // Lib does not support dynamically transitionStyle change
           transitionStyle="scroll"
-          showPageIndicator
+          showPageIndicator={this.state.isDotsVisible}
```

Comment:
Please move it to destructor at line 113 and use

`showPageIndicator={areDotsVisible}`